### PR TITLE
Updated og tags

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

**Comment by victorbnl** [comment]
If you merge my site this will become useless no? Because I deleted all files and regenerated a whole new site when doing it

**Comment by biodranik** [comment]
Why did you delete everything, instead of adding/modifying existing files? What was the reason?

**Comment by victorbnl** [comment]
Sorry if it annoys you, as the site I wanted to do had nearly no common points anymore I felt like it would be a lot easier to start all from the beginning

**Comment by biodranik** [comment]
It does not annoy me. I just don't understand the reason. "It was easier to throw everything out" sounds… strange. Every line of code and text in the current version was carefully designed and tested. Throwing it out without a good reason doesn't make any sense.

**Comment by victorbnl** [comment]
I thought it'd be easier but you're probably right

File: content/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Organic Maps: Offline Hike, Bike, Trails and Navigation"
-description: Fast detailed offline maps for travelers, tourists, hikers and cyclists, based on OpenStreetMap and curated with love by MapsWithMe founders.
+description: Fast detailed offline maps for travelers, tourists, hikers and cyclists, based on OpenStreetMap and curated with love by MapsWithMe (MapsMe) founders.
```
Comment (rtsisyk) [inline content/_index.md:3]: MAPS.ME is a registered trademark.

Comment (biodranik) [reply content/_index.md:3]: MapsMe too? Where did you check?

Comment (biodranik) [reply content/_index.md:3]: Am I breaking trademark rules if my CV shows that I am a founder and used to work in MAPSME?

**Review by vng — APPROVED** [review]

**Pull request merged**